src/main/java/com/google/devtools/build/remote/client/RemoteClientOptions.java

```diff
@@ -70,6 +92,27 @@
     public Path path = Paths.get("");
   }
 
+  @Parameters(
+    commandDescription = "Recursively downloads a OutputDirectory from remote cache.",
+    separators = "="
+  )
+  public static class GetOutDirCommand {
+    @Parameter(
+      names = "--digest",
```

Comment:
Suggestion: since all your commands involve digest, you could also make it the default parameter.
Also, add a "-o" shortname to the output name parameter wherever it exists.

Reply:
I had the same thought. Specifying the digest parameter everywhere is kind of repetitive, but there are plans to add commands (I am working on now) that don't take a digest so I think it should just be kept this way. 

I'll add the -o options for output paths.

Reply:
But even commands that don't require a digest could still have a default parameter. Up to you. In any case, I'd suggest adding the "-d" option for the digest as well.

Reply:
Sorry, I think I am misunderstanding what a default parameter is. Can you clarify? Thanks!

Reply:
Oh, I think you mean like this :http://jcommander.org/#_main_parameter. Looks good to me.

Reply:
Okay, so I tried this. There's a bug in my version (which is the latest maven release) of jcommander where main parameters don't support converters. This was fixed here: https://github.com/cbeust/jcommander/pull/390 but has not been released.

I can work around this by just using a string and converting later, or by trying to pull a later commit of jcommander from github instead of using maven. I'm not sure if either of these solutions are worth providing this small improvement to user interface. 

Any thoughts?

Reply:
Thanks for looking into this! It's not worth much trouble imho.
In the long term I'm hoping to cover most useful use cases so that the user wouldn't even need to call this directly :-)

Reply:
@ola-rozenfeld  Done. I added the -d flag, and -l for limits as a bonus.